packages/kit/src/constants.ts

```diff
@@ -25,6 +25,19 @@ export const CONTRACT_ADDRESSES = {
     FlowEVMBridgeConfig: "0x1e4aa0b87d10b141",
     FungibleTokenMetadataViews: "0xf233dcee88fe0abe",
   },
+  local: {
+    EVM: "0xf8d6e0586b0a20c7",
```

Comment:
Wouldn't it be less repetitive to go name then object of networks and also less likely to forget adding a contract name to a network that way?